src/pmCommand.py

```diff
@@ -122,23 +135,14 @@ def emptyline(self):
         pass
 
     def do_help(self, args):
+        """Displays the list of commands."""
         print("""pmCommand strikes back!
 
-Available commands:
- login              Provide password to get a new session at the appliance.
- logout             Logout from the appliance.
- sort               Toggle sorted output for pdu and outlet list
- exit               Exits from the application.
- help               Displays the list of commands.
- listipdus          Lists the IPDUs connected to the appliance.
- on                 Turns on outlets defined in <outlet list>.
- off                Turns off outlets defined in <outlet list>.
- cycle              Power cycles outlets defined in <outlet list>.
- lock               Locks outlets defined in <outlet list> in current state.
- unlock             Unlocks outlets define in <outlet list>.
- status             Displays status of outlets defined in <outlet list>.
- save               Saves configuration to flash.
-""")
+Available commands:""")
+
+        for name, func_obj in sorted(inspect.getmembers(self)):
+            if name.startswith('do_') and func_obj.__doc__:
+                print("  %-20s%s" % (name[3:], func_obj.__doc__))
```

Comment:
I should add some file with hints about [coding style](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file).

It will include:
* Preferred string formatting method in here is .format(